### PR TITLE
docs: update documentation for armadillo 3 to 4 update

### PR DESCRIPTION
**Pull request by marikaris**
Updated documentation for the armadillo 3 to 4 update after trying out the current manual.

### CODE REVIEW

File: docs/upgrade-3-4.md [hunk 1]
```diff
@@ -119,61 +119,15 @@ ls -ltr /usr/share/armadillo/application/
...
-then edit
+### 4.1 application.yml
 
+Edit the application.yml:
 ```bash
 nano /etc/armadillo/application.yml
 ```
+Add the following, below `docker-management-enabled: true`:
+`docker-run-in-container: false`
+
```
Comment (clemens-tolboom) [inline docs/upgrade-3-4.md:130]: This is not needed as we fixed that in https://github.com/molgenis/molgenis-service-armadillo/commit/c289e99aeacd721d4037a7e61db57321cf22b339

Maybe we should remove `docker-management-enabled: true` from template too ... or disable both there.

File: docs/upgrade-3-4.md [hunk 2]
```diff
@@ -119,61 +119,15 @@ ls -ltr /usr/share/armadillo/application/
...
 
-### 4.1 application template
-
-This step is optional, you can try if it works without. 
-
-The application settings could have new entries so you may need to check these.
-
-```bash
-# Change the version number v4.x.y
-wget https://raw.githubusercontent.com/molgenis/molgenis-service-armadillo/v4.x.y/application.template.yml
```
Comment (clemens-tolboom) [inline docs/upgrade-3-4.md:130]: I prefer they fetch the template file

File: docs/upgrade-3-4.md [hunk 3]
```diff
@@ -266,6 +219,81 @@ tail -f /var/log/armadillo/*
...
+Login into the website and go to the profiles tab. Here two profiles should be listed: `default` and `xenon`. 
+
+1. Stop the xenon profile. 
+2. Stop the default profile.
+3. Edit the default profile. 
+4. Change the "image" to `datashield/rock-base:latest` and save. 
+5. Start the default profile.
+6. Edit the "xenon" profile.
+7. Change the "image" to `datashield/rock-dolomite-xenon:latest` and save.
+8. Start the xenon profile.
```
Comment (clemens-tolboom) [inline docs/upgrade-3-4.md:235]: They are stopped and changed in stap 2.2 through UI and removed by docker commands.

File: docs/upgrade-3-4.md [hunk 4]
```diff
@@ -266,6 +219,81 @@ tail -f /var/log/armadillo/*
...
+Everything should be working fine now. You can try and login to your server via the central analysis server, using
+the `DSMolgenisArmadillo` (2.0.5 or up) package to test. 
+
+Enjoy =)
+Team Armadillo
 
-Enjoy. (team Armadillo)
+
+## Troubleshooting
+
```
Comment (clemens-tolboom) [inline docs/upgrade-3-4.md:245]: Nice ... maybe mention the section in step 1?

File: docs/upgrade-3-4.md (hunk 1 above)
Comment (marikaris) [reply docs/upgrade-3-4.md:132]: I've added this information to the top of step 1, I think it's good to give people options and be transparant :)

File: docs/upgrade-3-4.md (hunk 3 above)
Comment (marikaris) [reply docs/upgrade-3-4.md]: You're right! I've changed it

File: docs/upgrade-3-4.md (hunk 4 above)
Comment (marikaris) [reply docs/upgrade-3-4.md]: Great suggestion. Done :)

**Review by clemens-tolboom — APPROVED** [review]
LGTM

**Pull request merged**